channels-rke2.yaml

```diff
@@ -325,6 +325,33 @@ releases:
       rke2-canal:
         repo: rancher-rke2-charts
         version: v3.21.4-build2022020801
+  - version: v1.21.10-rc2+rke2r2
+    minChannelServerVersion: v2.6.1-alpha1
```

Comment:
I think this needs to be updated to be `v2.6.4-alpha1` per our last internal KDM discussion